install_process_test.go

```diff
@@ -180,7 +175,6 @@ func testInstallProcess(t *testing.T, context spec.G, it spec.S) {
 				"get",
 				"yarn-offline-mirror",
 			}))
-			Expect(executions[0].Env).To(ContainElement(MatchRegexp(fmt.Sprintf(`^PATH=.*:%s/bin:node_modules/.bin$`, yarnLayerPath))))
 
```

Comment:
```suggestion
Expect(executions[0].Env).To(ContainElement(MatchRegexp(`^PATH=.*:node_modules/.bin$`)))
```
Don't we still need to assert that "node_modules/.bin" is added to path?

Reply:
good call! Not sure why we deleted these, must have been an accident!